### PR TITLE
Publish pipeline issue

### PR DESCRIPTION
**Pull request by francis-whitesell-earnest**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,5 +33,8 @@
   "files": [
     "index.js",
     "mocha.js"
-  ]
+  ],
+  "dependencies": {
```
Comment (mjacksonest) [inline package.json]: This should be devDependencies or it will get installed along with the package

Comment (francis-whitesell-earnest) [reply package.json]: I removed this because it was unneeded. Thanks!

File: Dockerfile [hunk 2]
```diff
@@ -3,10 +3,9 @@ FROM node:10
 ARG NPM_TOKEN  
 COPY .npmrc .npmrc  
 COPY package.json package.json
-RUN npm install autopublish
-RUN ./node_modules/.bin/autopublish .
+COPY . .
```
Comment (mjacksonest) [inline Dockerfile]: Sorry I missed this earlier, without .dockerignore file, you'll pickup undesirables like .git folder https://docs.docker.com/engine/reference/builder/#dockerignore-file

Comment (mjacksonest) [reply Dockerfile]: Confirmed my memory was correct:
```$ cat Dockerfile
FROM node:10
WORKDIR /tmp
COPY . .


$ docker build -t test .
Sending build context to Docker daemon  37.89kB
Step 1/3 : FROM node:10
 ---> ea119cebc1c3
Step 2/3 : WORKDIR /tmp
 ---> Using cache
 ---> bbea7e2e4556
Step 3/3 : COPY . .
 ---> Using cache
 ---> e3593b2df4fc
Successfully built e3593b2df4fc
Successfully tagged test:latest

$ docker run -it test ls -al
total 16
drwxrwxrwt 1 root root 4096 Jan  3 01:15 .
drwxr-xr-x 1 root root 4096 Jan  3 01:16 ..
drwxr-xr-x 6 root root 4096 Jan  3 01:15 .git
-rw-r--r-- 1 root root   36 Jan  3 01:15 Dockerfile

$ ls -al
total 8
drwxr-xr-x  4 michaeljackson  staff  128 Jan  2 18:15 .
drwxr-xr-x  7 michaeljackson  staff  224 Jan  2 14:27 ..
drwxr-xr-x  9 michaeljackson  staff  288 Jan  2 18:16 .git
-rw-r--r--  1 michaeljackson  staff   36 Jan  2 18:15 Dockerfile
```
Ultimately, not an issue in this case other than slowing down build copying unneeded files.

Comment (francis-whitesell-earnest) [reply Dockerfile]: added .dockerignore

**Pull request merged**